src/Momento.Sdk/Config/Configurations.cs

```diff
@@ -163,5 +163,33 @@ public static IConfiguration V1(ILoggerFactory? loggerFactory = null)
                 return new LowLatency(finalLoggerFactory, retryStrategy, transportStrategy);
             }
         }
+
+        /// <summary>
+        /// This config optimizes for lambda environments. In addition to the low latency settings of
+        /// <see cref="LowLatency"/>, this configures the clients to eagerly connect to the Momento service
+        /// to avoid the cold start penalty of establishing a connection on the first request.
+        /// </summary>
+        public class Lambda : Configuration
+        {
+            private Lambda(ILoggerFactory loggerFactory, IRetryStrategy retryStrategy, ITransportStrategy transportStrategy)
+                : base(loggerFactory, retryStrategy, transportStrategy)
+            {
+
+            }
+
+            /// <summary>
+            /// Provides the latest recommended configuration for a lambda environment.
+            /// </summary>
+            /// <param name="loggerFactory"></param>
+            /// <returns></returns>
+            public static IConfiguration Latest(ILoggerFactory? loggerFactory = null)
+            {
+                var config = LowLatency.V1(loggerFactory);
```

Comment:
I think maybe you should go with `InRegion.Default` rather than `InRegion.LowLatency`.  I don't think lambdas are guaranteed to have great networking traits so making the timeout a tiny bit more lenient is probably the way to go.

Sorry for not mentioning sooner.

@eaddingtonwhite can you weigh in on this?

Reply:
I changed this to `InRegion.Default` [here](https://github.com/momentohq/client-sdk-dotnet/pull/427/commits/b164b6a621a1420c8c183d93545883a3c341af16)

Reply:
Agree w/ this. Start more conservative w/ default timeouts we can always make more agressive down line.

Reply:
OK. In that case https://github.com/momentohq/client-sdk-dotnet/pull/427/commits/b164b6a621a1420c8c183d93545883a3c341af16 implements this.